### ISSUE TITLE
Tags Not Getting Wrapped - Causing Collector Failure

### ISSUE THREAD

**Issue by johnpierson** [DynamoDS/DynamoRevit#2742]
## Dynamo version

2.6.0

## Revit version

2021

## Operating system

Windows 10

## What did you do? 

Try to get electrical tags with Categories > All Elements of Category

## What did you expect to see?

The electrical tag elements

## What did you see instead?

An error:
![image](https://user-images.githubusercontent.com/15744724/130268679-d13f0a21-d779-4bc4-afdb-a1bda888e86f.png)

I wanted to see something similar to what Spring Nodes does. (However, even the resulting elements from that node fail on down the script because they are wrapped as unknown elements)
![image](https://user-images.githubusercontent.com/15744724/130268636-85f2e334-c87d-4317-b94d-93dfb61bf05a.png)

**Comment by ZiyunShang** [comment]
Hi @johnpierson , I just try this category 'Electrical Equipment Tags' on my side, my Revit is 2022, and Dynamo version is 2.12.0. The workflow works fine.
![Electrical Equipment Tags](https://user-images.githubusercontent.com/33445445/147625122-8c0df42d-25da-415f-b16f-91f7def971ea.PNG)
Are you still experiencing this issue now? If yes, could you share your sample file and we can reproduce this issue on our side.

**Comment by johnpierson** [comment]
Yep, this appears to be fixed in 2022 now for these categories. 3d view templates still are not wrapped, but I think that might be a separate issue.
